app/pathogen/arbovirus/wip/page.tsx

```diff
@@ -0,0 +1,39 @@
+"use client"
+import { useScrollSectionGroup } from "@/components/ui/scroll-section-group/use-scroll-section-group";
+import { notFound, useSearchParams } from "next/navigation";
+import React, { useEffect } from "react";
+import { redesignedArbovirusPageSections, sectionUrlParameterToSectionId } from "./sections";
+
+export default function RedesignedArbovirusPage() {
+  const searchParams = useSearchParams();
+
+  const { renderScrollSectionGroup, moveScrollSectionGroupToSection } = useScrollSectionGroup({
+    scrollSectionGroupProps: {
+      sections: redesignedArbovirusPageSections,
+      className:"col-span-10 row-span-2",
+      scrollThrottleThresholdMilliseconds: 700
+    }
+  })
+
+  useEffect(() => {
+    const sectionUrlParameter = searchParams.get('section');
+    const sectionId = sectionUrlParameterToSectionId({ urlParam: sectionUrlParameter })
+
+    if(sectionId) {
+      moveScrollSectionGroupToSection({ sectionId });
+    }
+  }, [searchParams])
+
+  if(process.env.NEXT_PUBLIC_WEBSITE_REDESIGN_ENABLED !== 'true') {
+    return notFound();
+  }
+  
+  return (
+    <div className="col-span-12 row-span-2 grid gap-0 grid-cols-12 grid-rows-2 grid-flow-col w-screen overflow-hidden border-box -my-4 -ml-4">
```

Comment:
Silly negative margin thing to fight the layout. Yeah I know it's silly, it's the simplest solution to the padding on the layout please bear with me lol